perceval/backends/core/confluence.py

```diff
@@ -279,6 +291,12 @@ def setup_cmd_parser():
         parser.parser.add_argument('url',
                                    help="URL of the Confluence server")
 
+        # Optional arguments
+        group = parser.parser.add_argument_group('Confluence page options')
+        group.add_argument('--add_ancestors', dest='add_ancestors',
+                           nargs='+', type=bool, default=False,
```

Comment:
why `nargs='+'` ?

Reply:
No idea? :-) I thought that was what made the argument optional. I understand from the comment above that this block will be removed, right?

Reply:
ehehe
yes, it guess it's better to remove it.